### PR TITLE
add config for v0.10.11 src14 dec2276

### PR DESCRIPTION
**Pull request by rlopezcoto**
# New Prod config

Self check-list:

- [x] I have checked the lstchain config, in particular for:
  - [x] `az_tel` instead of `sin_az_tel` if data to be analyzed have been produced with lstchain <= v0.9.7
  - [x] "increase_nsb" and "increase_psf" are provided in "image_modifier" (if used)
- [x] I have checked the environment in the lstmcpipe config and it is the one used to analyse DL>1 data
- [x] I have provided the command (in README), or script (in additionnal .py file) used to produce the lstmcpipe config

## Prod_ID
20240821_v0.10.11_src14_tuned

## Short description of the config
full DL1ab production (with NSB tuning for src14) with lstchain v0.10.11

## Why this config is needed 
Analysis of future data taken on src13

### CODE REVIEW

File: production_configs/20240821_v0.10.11_src14_dec2276_tuned/lstmcpipe_config_2024-08-21_PathConfigAllSkyFullDL1ab.yaml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+stages_to_run:
+- dl1ab
+- merge_dl1
+- train_pipe
+- dl1_to_dl2
+- dl2_to_irfs
+stages:
+  dl1ab:
+  - input: 
+      /fefs/aswg/data/mc/DL1/AllSky/20240821_v0.10.11_src14_dec2276_tuned/TrainingDataset/dec_2276/GammaDiffuse/node_corsika_theta_66.446_az_75.983_
```
Comment (vuillaut) [inline production_configs/20240821_v0.10.11_src14_dec2276_tuned/lstmcpipe_config_2024-08-21_PathConfigAllSkyFullDL1ab.yaml:26]: Hi Ruben
Thanks for the PR.

Some issues should be addressed. This is a dl1ab production, but the starting point (DL1 input) does not exist (the config here would start from `/fefs/aswg/data/mc/DL1/AllSky/20240821_v0.10.11_src14_dec2276_tuned/` but that does not exist yet).
The mistake comes from the source prod ID in the command. You could start from prod `20240131_allsky_v0.10.5_all_dec_base` as base one and the command would be:

```
lstmcpipe_generate_config PathConfigAllSkyFullDL1ab --dec_list dec_2276 --prod_id 20240821_v0.10.11_src14_dec2276_tuned --kwargs source_prod_id=20240131_allsky_v0.10.5_all_dec_base
```

**Review by vuillaut — CHANGES_REQUESTED** [review]

File: production_configs/20240821_v0.10.11_src14_dec2276_tuned/lstmcpipe_config_2024-08-21_PathConfigAllSkyFullDL1ab.yaml (hunk 1 above)
Comment (rlopezcoto) [reply production_configs/20240821_v0.10.11_src14_dec2276_tuned/lstmcpipe_config_2024-08-21_PathConfigAllSkyFullDL1ab.yaml]: ups, apologies! Thanks for the correct kwargs, I updated the files and checked that the input folders exist

**Review by vuillaut — APPROVED** [review]
looks good

**Comment by vuillaut** [comment]
I just realised that you probably want to use lstchain-v0.10.11 not lstchain-v0.10.7 as stated in the config.
I will update the config and start the prod

**Comment by vuillaut** [comment]
This prod is done @rlopezcoto 🏁

**Pull request merged**